### PR TITLE
Add admin unit search web service

### PR DESCRIPTION
**Pull request by elemoine**
This PR adds a JSON web service for searching admin units, and make the search field rely on that search service.

The URL of the service is `/adminunit` and the `q` parameter is used for searching admin units. For example: `/adminunit?q=mor`.

### CODE REVIEW

File: thinkhazard/models.py [hunk 1]
```diff
@@ -18,10 +17,50 @@
...
+
+    def __json__(self, request):
+        return {'admin0': self.admin0_name, 'admin1': self.name}
+
+
+class Admin2(Admin):
+    __tablename__ = 'admin2'
+    order = 2
+
+    admin0_name = Column(Unicode)
```
Comment (ingenieroariel) [inline thinkhazard/models.py]: I agree this denormalization (admin0 field on admin2) is a good idea for performance reasons. Is there a need for a constraint or a trigger at the database level to ensure it stays in sync?

Comment (elemoine) [inline thinkhazard/models.py]: The input data was already denormalized in that respect.

Comment (fvanderbiest) [inline thinkhazard/models.py]: We just took the source data "as they are".

If we were to do joins, we would need working foreign keys, hence uniqueness on GAUL codes ... which we do not have ATM.

**Comment by ingenieroariel** [comment]
:+1:

**Comment by elemoine** [comment]
@ingenieroariel We're going to update the demo server, and send a link to the group.

**Pull request merged**